client/cs_statefile.cpp

```diff
@@ -294,9 +294,13 @@ int CLIENT_STATE::parse_state_file_aux(const char* fname) {
             }
             if (avp->missing_coproc) {
                 msg_printf(project, MSG_INFO,
-                    "Application uses missing %s GPU",
+                    "App version uses missing GPU '%s' - discarding",
```

Comment:
Is it really needed to show `discarding` for all cases, not only when Apple GPU is missing?
It might be a little bit misleading for the users.